test/Controller/SubjectControllerTest.php

```diff
@@ -43,6 +43,9 @@ public function it_has_metadata()
         $this->assertSame('https://www.example.com/iiif/ban%2Fner/full/full/0/default.jpg', $crawler->filter('meta[property="og:image"]')->attr('content'));
         $this->assertSame('1800', $crawler->filter('meta[property="og:image:width"]')->attr('content'));
         $this->assertSame('1600', $crawler->filter('meta[property="og:image:height"]')->attr('content'));
+        $this->assertSame('application/rss+xml', $crawler->filter('link[rel="alternate"]')->attr('type'));
+        $this->assertSame('eLife Sciences Subject recent articles', $crawler->filter('link[rel="alternate"]')->attr('title'));
+        $this->assertSame('/rss/recent-subject.xml', $crawler->filter('link[rel="alternate"]')->attr('href'));
```

Comment:
@Maelplaine have taken on your suggestion of dropping the query parameter and adding the subject id into the path of the rss feed. This would just prefix the subject id with `/rss/recent-`. What do you think?

Reply:
Once we agree on the url structure, I will amend the PR for `journal-formula`: https://github.com/elifesciences/journal-formula/pull/131

Reply:
@Maelplaine I have implemented the agreed `/rss/subject/{id}.xml`